src/components/shell/shell.js

```diff
@@ -219,12 +219,19 @@ class Shell extends Component {
    */
   componentDidMount() {
```

Comment:
remove the listeners on `componentWillUnmount`

Reply:
automatically made by https://github.com/kaltura/playkit-js-ui/blob/master/src/event/with-event-manager.js#L20